.gitpod.yml

```
@@ -1,7 +1,10 @@
 image: nfcore/gitpod:latest
 tasks:
-  - name: install current state of nf-core/tools
-    command: python -m pip install --upgrade -r requirements-dev.txt -e .
+  - name: install current state of nf-core/tools and setup pre-commit
+    command: |
+      python -m pip install -e .
+      python -m pip install -r requirements-dev.txt
```

```suggestion
      python -m pip install --upgrade -r requirements-dev.txt -e .
```
This worked before 🙂